### PR TITLE
Fix jwt auth

### PR DESCRIPTION
**Pull request by AsterNighT**
Signed-off-by: AsterNighT <klxjt99@outlook.com>

### CODE REVIEW

**Comment by futuretech6** [comment]
Not quite understand, so we are not using cookie?

**Comment by AsterNighT** [comment]
Check this http://www.ruanyifeng.com/blog/2018/07/json_web_token-tutorial.html

> Not quite understand, so we are not using cookie?

File: pkg/account/handler.go [hunk 1]
```diff
@@ -173,48 +169,18 @@ func (h *AccountHandler) LoginAccount(c echo.Context) error {
...
 }
 
-// @Summary logout using cookie
-// @Description
-// @Tags Account
-// @Produce json
-// @Success 200 {string} api.ReturnedData{data=nil}
-// @Failure 400 {string} api.ReturnedData{data=nil}
-// @Router /account/logout [POST]
-func (h *AccountHandler) LogoutAccount(c echo.Context) error {
```
Comment (futuretech6) [inline pkg/account/handler.go:204]: Account cannot be logged out until another user is logged in?

Comment (AsterNighT) [reply pkg/account/handler.go:204]: If you want to logout, simply drop the token in the frontend

**Review by futuretech6 — APPROVED** [review]

**Pull request merged**